src/main/java/athleticli/parser/Parser.java

```diff
@@ -53,6 +53,7 @@
  * Defines the basic methods for command parser.
  */
 public class Parser {
+    private static final String INVALID_YEAR = "0000";
```

Comment:
Can you specify this boundary / exception in the UG as well?

Reply:
Explicitly mentioning this "0000" could makes things unnecessarily detailed since in practice, no user is going to use this year. We can mention that it needs to be a valid date.

Reply:
My bad, thought there is a year 0, so invalid is fine!